components/registry-facade/pkg/registry/metrics.go

```diff
@@ -68,8 +69,22 @@ func newMetrics(reg prometheus.Registerer) (*metrics, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	blobDownloadSpeedHist := prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    "blob_req_bytes_second",
```

Comment:
If it means blob download speed, does it make sense to rename the metric with that?
Maybe `gitpod_registry_facade_download_speed_bytes_per_second`